package.exs

```diff
@@ -3,16 +3,16 @@ defmodule GenRPC.Mixfile do
 
   def project do
     [app: :gen_rpc,
-     version: "2.0.0",
+     version: "2.1.0",
      description: "A scalable RPC library for Erlang-VM based languages",
      package: package]
   end
 
   defp package do
     [files: ~w(include src LICENSE Makefile package.exs README.md TODO.md CHANGELOG.md rebar.config rebar.config.script otp-release.escript),
-     maintainers: ["Panagiotis PJ Papadomitsos"],
```

Comment:
I think this should be Ok in the context of our maintenance.

Reply:
✅